modules/test/playwright/pages/users-admin-web/EditUserPage.ts

```diff
@@ -85,5 +91,19 @@ export class EditUserPage {
 				strictEqual
 			);
 		};
+		this.screenNameInput = page.getByLabel("Screen Name");
+		this.emailAddressInput = page.getByLabel("Email Address");
+		this.saveButton = page.getByRole('button', {name: 'Save'});
+		this.passwordConfirmationFrame = page.frameLocator('iframe[title="Confirm Password"]')
+		this.yourPasswordInput = this.passwordConfirmationFrame.getByLabel('Your Password')
+		this.confirmButton = this.passwordConfirmationFrame.getByRole('button', {name: 'Confirm'});
+	}
+
+	async updateUser(screenName: string, emailAddress: string) {
+		await this.screenNameInput.fill(screenName);
+		await this.emailAddressInput.fill(emailAddress);
+		await this.saveButton.click();
+		await this.yourPasswordInput.fill('test');
+		await this.confirmButton.click();
```

Comment:
can you move these operations directly in the test? I don't think this method will be used so often because we are only changing screenName and email